### PR TITLE
style text fields

### PR DESCRIPTION
**Pull request by KacperKoza343**
## Related issues
[Keywords for linking issues](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests)

### CODE REVIEW

File: packages/apps/human-app/frontend/src/styles/theme.ts [hunk 1]
```diff
@@ -93,6 +93,20 @@ export const theme: ThemeOptions = {
         },
       },
     },
+    MuiChip: {
```
Comment (MicDebBlocky) [inline packages/apps/human-app/frontend/src/styles/theme.ts]: Check is necessary if custom MuiSvgIcon

File: packages/apps/human-app/frontend/src/components/data-entry/select.tsx [hunk 2]
```diff
@@ -30,26 +29,22 @@ export function Select({
     <Controller
       name={name}
       render={({ field, fieldState }) => (
-        <FormControl
-          error={Boolean(fieldState.error)}
```
Comment (MicDebBlocky) [inline packages/apps/human-app/frontend/src/components/data-entry/select.tsx:34]: error={Boolean(fieldState.error)} shoud exist in component

File: packages/apps/human-app/frontend/src/components/data-entry/select.tsx [hunk 3]
```diff
@@ -30,26 +29,22 @@ export function Select({
...
+            label={label}
           >
             {options.map((elem) => (
-              <MenuItem key={elem.id} value={elem.value}>
+              <MenuItem key={crypto.randomUUID().toString()} value={elem.value}>
                 {elem.name}
               </MenuItem>
             ))}
           </SelectMui>
-          <FormHelperText>{fieldState.error?.message}</FormHelperText>
```
Comment (MicDebBlocky) [inline packages/apps/human-app/frontend/src/components/data-entry/select.tsx]: shoud exist in component

File: packages/apps/human-app/frontend/src/styles/theme.ts (hunk 1 above)
Comment (KacperKoza343) [reply packages/apps/human-app/frontend/src/styles/theme.ts]: not necessary, removed

File: packages/apps/human-app/frontend/src/components/data-entry/select.tsx (hunk 2 above)
Comment (KacperKoza343) [reply packages/apps/human-app/frontend/src/components/data-entry/select.tsx:34]: that should not be removed

File: packages/apps/human-app/frontend/src/components/data-entry/select.tsx (hunk 3 above)
Comment (KacperKoza343) [reply packages/apps/human-app/frontend/src/components/data-entry/select.tsx]: that should not be removed

**Pull request merged**